packages/lesswrong/components/questions/AnswersList.tsx

```diff
@@ -48,7 +53,7 @@ const AnswersList = ({post, classes}: {
 
   if (results && results.length) {
     return <div className={classes.root}>
-      <SectionTitle title={<span>{ results.length } Answers</span>}/>
+      <SectionTitle title={<span>{ results.length } Answers, sorted by <Components.AnswersSorting post={post}/></span>}/>
```

Comment:
The more recently idiomatic style of this codebase is to destructure components off the components object, rather than accessing it directly. See 4 lines above.

Reply:
In my screenshot I added a span here.